### ISSUE TITLE
`getmac -4 ip` doesn't return result in Mac OS X Mojave

### ISSUE THREAD

**Issue by paolo-tanium** [GhostofGoes/getmac#33]
**Describe the bug**
When you run `getmac -4 192.168.1.252` (or any valid ip address), the macaddress is not printed out as a result. 

**To Reproduce**
Using Mac OS X Mojave, run:

```bash
> python2.7 -m getmac -4 192.168.1.252
> 
```

Returns with no output.

If you run it with the `--debug` flag:

```bash
python2.7 -m getmac -4 192.168.1.252 --debug --no-network-request
Trying: 'cat /proc/net/arp'
Exception: Command '['/bin/cat', '/proc/net/arp']' returned non-zero exit status 1
Trying: '<lambda>' (to_find: '192.168.1.252')
Exception: [Errno 2] No such file or directory
Trying: 'arp 192.168.1.252'
Result: 28:cf:e9:1a:e7:17

Trying: 'arp -an'
Result: 28:cf:e9:1a:e7:17

Trying: 'arp -an 192.168.1.252'
Exception: Command '['/usr/sbin/arp', '-an', '192.168.1.252']' returned non-zero exit status 1
Trying: 'arp 192.168.1.252'
Result: 28:cf:e9:1a:e7:17

Trying: 'arp -a'
Result: 28:cf:e9:1a:e7:17

Trying: 'arp -a 192.168.1.252'
Exception: Command '['/usr/sbin/arp', '-a', '192.168.1.252']' returned non-zero exit status 1
Trying: '_uuid_ip' (to_find: '192.168.1.252')
Result: None

Trying: '_scapy_ip' (to_find: '192.168.1.252')
Exception: No module named scapy.layers.l2
Trying: '<lambda>' (to_find: '192.168.1.252')
Exception: No module named arpreq
Raw MAC found: None
```

So it seems to find the mac address through one of its methods, but it doesn't report it to the CLI. I tried this programmatically as well, and the call to `get_mac_address` returned `None`.

**Expected behavior**
The mac address should have been reported to CLI/function caller.

**System info**
(please complete the following information):
 - OS name: Mac OS X
 - OS Version: Mojave
 - Python version: [2.7.10 x64]

**Additional context**
Only tested with Mojave, but I'd be curious if it were happening on other OSes as well.

**Comment by GhostofGoes** [comment]
What version of getmac are you using?

**Comment by GhostofGoes** [comment]
I'm not able to reproduce this on Windows with the same command and Python version with the last PyPI release (0.6.0).

There are a number of improvements to OSX in the upcoming release. Could you try installing from the repo and seeing if the problem reoccurs?

```bash
python2.7 -m pip uninstall getmac
python2.7 -m pip install https://github.com/ghostofgoes/getmac/archive/master.tar.gz
python2.7 -m getmac -4 192.168.1.252
```

**Comment by paolo-tanium** [comment]
> What version of getmac are you using? 

Using version getmac 0.6.0. 

> Could you try installing from the repo and seeing if the problem reoccurs?

Same behavior. Didn't return anything.

**Comment by GhostofGoes** [comment]
Could you run the command again for the version installed from the repo with extra debugging output (`-ddd`)?

```python
python2.7 -m getmac -4 192.168.1.252 -ddd
```

Also, are you running with the system python, or a Python you installed from elsewhere (brew, etc.)?

Apologies for making you run things. Going to set back up the one Mac I have (2010 Mac Air) and see if I can get to the root of this.

**Comment by jos-b** [comment]
I'm on macOS Mojave here and I've just installed Python 2.7 (from pyenv).

I'm getting an IP address back.
```py
➜ python2.7 -m getmac -4 192.168.10.101
[mac address back]
```

Python information:
```
➜ python2.7   
Python 2.7.2 (default, Jan 28 2019, 17:03:55) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.0 (clang-1000.10.44.4)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import getmac
>>> getmac.__version__
'0.7.0'
```

**Comment by GhostofGoes** [comment]
@paolo-tanium I just released a new version, you should give that a try. It worked on High Sierra with the system Python (though not pip installed due to SSL issues since the system Python is so ancient).

**Comment by GhostofGoes** [comment]
Not able to reproduce and some potential causes have been addressed in recent releases (0.7.0 and 0.8.0) so I'm going to close this. If it's still happening, please comment and I'll reopen the issue.
